convert_rmd_file.py

```diff
@@ -42,8 +42,8 @@ def _sanitize_rmd(data: str) -> bytes:
     lines = data.split('\n')
     sanitized_file = []
     for line in lines:
-        if line.startswith('```'):
-            sanitized_line = '```'
+        if line.startswith('`'):
```

Comment:
Just reading the Rmd spec…

I guess the original code would sanitize code blocks like: 
```
```{r}
print("hello")
```
by removing the `{r}`, so it would keep the code block but it wouldn't be executable.

I'm wondering how it handles single-line code chunks like:
```
`r system(curl("https://"))`
```
(which is the example in the [report](https://drive.google.com/file/d/1twrR6UjiVVrpEs-Yf0zO9cTC4lg6krx5/view)).

I think this code would just replace the line with a single backtick which isn't exactly correct, right?

Reply:
Maybe what we want is:
1. Replace `triplebacktick{r}` with just `triplebacktick` (as before)
2. Repace `singlebacktickr line` with just `singlebacktick line`

Does that handle all cases? 🤔

Reply:
ah yes good point, I can replace it with a the enclosed string